helix-core/src/main/java/org/apache/helix/ConfigAccessor.java

```diff
@@ -604,7 +604,47 @@ public CloudConfig getCloudConfig(String clusterName) {
       return null;
     }
 
-    return new CloudConfig.Builder(record).build();
+    return new CloudConfig(record);
+  }
+
+  /**
+   * Delete cloud config fields (not the whole config)
+   * @param clusterName
+   * @param cloudConfig
+   */
+  public void deleteCloudConfigPatrial(String clusterName, CloudConfig cloudConfig) {
```

Comment:
Shall we just call it deleteCloudConfigFields?

Reply:
Changed to deleteCloudConfigFields.